### PR TITLE
RD-5709 modify ts-migration-progress

### PR DESCRIPTION
**Pull request by PatrycjaKubiczek**
## Description

RD-5709
- in https://github.com/cloudify-cosmo/cloudify-ui-components/pull/154 I decided to use `ts-migrate` which adds ts-expected-error instead of ts-nocheck, that's why I needed to adjust the script which calculates our TS migration

## Screenshots / Videos

n/a

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

CI checks passed ✅ 

## Documentation

n/a

### CODE REVIEW

**Comment by Vorbert-Kruk** [comment]
It's worth to mention, that with that approach we'd reduce progress on specific repositories
E.g. Currently on [stage repository](https://github.com/cloudify-cosmo/cloudify-stage) we are having 9 files which are having 
`@ts-expect-error` declaration, without `@ts-nocheck`

So with that approach, we'd consciously decrease typescript migration on this repo
(I've quickly checked `composer` and `topology` and it seems that there are no such cases here)

I'm not against it, but just wanted to mention the consequences of the following modifications, so that everyone would be aware of them 😁 
(if we'd go with such an approach, it would be worth to mention it on a weekly, while going through typescript migration progress 😉)

WDYT about this subject @qooban?

**Review by qooban — COMMENTED** [review]
Added one comment.

@Vorbert-Kruk 

> I'm not against it, but just wanted to mention the consequences of the following modifications, so that everyone would be aware of them grin

Yep, we discussed that with Patrycja before she created this PR. I said it's fine that we will have decrease in TS migration status in `cloudify-stage` (I did the same check as you and only `cloudify-stage` is affected).

> (if we'd go with such an approach, it would be worth to mention it on a weekly, while going through typescript migration progress wink)

Good point. Note in UI weekly added.

**Review by Vorbert-Kruk — DISMISSED** [review]
Looks good!
I don't have any additional comments

Thanks @qooban for providing a background info 💪

File: scripts/get-ts-migration-progress.sh [hunk 1]
```diff
@@ -24,10 +24,10 @@ TS_FILES=`find $DIRECTORIES_TO_CHECK -type f \( -name '*.ts' -o -name '*.tsx' \)
 TS_FILES_COUNT=`echo "$TS_FILES" | sed -r '/^\s*$/d' | wc -l`
 echo "Found $TS_FILES_COUNT TS/TSX files"
 if test $TS_FILES_COUNT -gt 0; then
-    TS_NOCHECK_COUNT=`grep '@ts-nocheck' $TS_FILES | wc -l || true`
-    echo "$TS_NOCHECK_COUNT out of these TS/TSX files have a '@ts-nocheck' header"
+    TS_NON_FULLY_MIGRATED_FILES_COUNT=`grep -E '@ts-nocheck|@ts-expect-error' $TS_FILES | wc -l || true`
```
[the remark this answers is not in the document]
Comment (PatrycjaKubiczek) [reply scripts/get-ts-migration-progress.sh]: yes, you were correct :) adjusted code: https://github.com/cloudify-cosmo/cloudify-ui-common/pull/135/commits/853e8eed48e6fb9c8fe06821eaed20ba76c4f1a5

**Review by qooban — APPROVED** [review]

**Pull request merged**